docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -38,6 +78,24 @@ kubectl apply --filename serviceaccount.yaml
    or use a different `Service Account`, you need to modify `k8s-events.yaml`
    accordingly.
 
+   ```yaml
+apiVersion: sources.eventing.knative.dev/v1alpha1
+kind: ApiServerSource
+metadata:
+  name: testevents
+  namespace: default
+spec:
+  serviceAccountName: events-sa
+  mode: Resource
+  resources:
+  - apiVersion: v1
+    kind: Event
+  sink:
+    apiVersion: eventing.knative.dev/v1alpha1
+    kind: Broker
+    name: default
+```
```

Comment:
New line at the end of the code back, and new line between the text and the start of the next code block. (New lines added between 97 & 98 and 98 & 99.)